### ISSUE TITLE
Are bookmarks a pdf accessibility feature?

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#5]
I have a strong suspicion that the "bookmark" value for accessibility feature was intended for PDFs, which is more of a table of contents. There's a [WCAG technique](https://www.w3.org/WAI/WCAG21/Techniques/pdf/PDF2.html) about adding them, after all, and I don't recall promoting this for ebooks when we created the vocabulary.

Putting aside the fact we don't have a common means of distributing the kind of digital "dog-ear" bookmarks reading systems provide, is there a special accessibility case for them even if we could? They can be generally useful for all users, but what would be the scenario where they add accessibility that is missing? I can't imagine professors distribute dog-eared copies of print course material to their students, for example.

Anyway, if this is a PDF feature, it would be good to clearly label it as such and equate it with a table of contents.

**Comment by mattgarrish** [comment]
I'm probably 99%+ certain this was added by Chuck for PDFs.

What I would question is do we need this at all @madeleinerothberg @clapierre ?

Should we add a note in the table of contents definition that the value can also be used for a toc created using PDF's bookmarks feature rather than have a second format-specific name for the same thing?

I be surprised if there's been any uptake of this metadata for PDF, so losing the value in this case doesn't seem problematic. We can always revisit "bookmarks" when there's a better case for them.

**Comment by madeleinerothberg** [comment]
Another possibility is that this is a reading system feature that tells users that this reading tool will allow you to place bookmarks. But we haven't had much discussion about reading systems offering this markup, and also it's likely that every reading system does offer bookmarks. If you'd like to deprecate it and add the note to the TOC definition, that makes sense to me.
